radix-engine-tests/Cargo.toml

```diff
@@ -37,8 +37,9 @@ walkdir = { version = "2.3.3" }
 paste = { version = "1.0.13" }
 lazy_static = { version = "1.4.0" }
 hex = { version = "0.4.3", default-features = false }
-tuple-return = { path = "./tests/blueprints/tuple-return", features = ["test"], default-features = false }
+tuple-return = { path = "./assets/blueprints/tuple-return", features = ["test"], default-features = false }
```

Comment:
It's weird that we have a dependency on the test blueprint. Possible to remove it?

Reply:
This dependency is needed for one test which calls directly method from the `tuple-return` blueprint. It would be possible to move that test to separate project or even add it as a test inside this specific blueprint, but this will require manually to run that one test (or use additional script) and will need CI script update also. So I am not sure if the juice is worth the squeeze however I have working change (with moving test to blueprint). @iamyulong what do you think?

Reply:
Moved `tuple-return` test to `scrypto-test` project.